static_frame/core/batch.py

```diff
@@ -951,6 +953,455 @@ def shift(self,
                 fill_value=fill_value,
                 )
 
+    # ---------------------------------------------------------------------------
+    # na handling
+    def isna(self) -> "Batch":
```

Comment:
Please use single strings.

```suggestion
    def isna(self) -> 'Batch':
```